packages/ui/react-ui-searchlist/src/components/SearchList.tsx

```diff
@@ -139,7 +140,7 @@ const SearchListItem = forwardRef<HTMLDivElement, SearchListItemProps>(
         {...props}
         onSelect={handleSelect}
         className={mx(
-          'p-1 rounded select-none cursor-pointer data-[selected]:bg-neutral-450/10 data-[selected]:hover:bg-25/10',
+          'p-1 select-none cursor-pointer data-[selected]:bg-neutral-450/10 data-[selected]:hover:bg-25/10',
```

Comment:
Please don’t remove `rounded`, this is a highlightable element, and all such elements in our design system are rounded, even currentable table rows.

Reply:
If the idea is to remove rounding on highlightable elements, we should/must apply the same treatment to the items in Select, DropdownMenu, Tree, Table, and possibly others that I’m not recalling.

Reply:
is that true also for Select options?

Reply:
anyway, moving to a different PR...